CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ project ("AprilTagTrackers")
 
 # Include sub-projects.
 add_subdirectory("wxWidgets")
-add_subdirectory ("AprilTagTrackers")
+add_subdirectory("AprilTagTrackers")
+add_subdirectory("apriltag" EXCLUDE_FROM_ALL)

```

Comment:
I think `EXCLUDE_FROM_ALL` should be removed because `apriltag` is a necessary component.

Reply:
Nope. `EXCLUDE_FROM_ALL` prevents every single target from `apriltag` being built, which includes the examples. We only need the library, which is marked as a dependency at line 17: `add_dependencies("AprilTagTrackers" "apriltags")`. This tells CMake that the target `apriltags`, which is what builds the library, is depended upon by AprilTagTrackers, and must be built first. This should speed up build time, as we don't waste time building targets that we don't use. Also, I had errors when building the example targets on my system, which not building them prevents.

Reply:
Ah, ok. I disabled them explicitly in my branch.